docs/Agent/user-guide/guides/end-to-end-workflow.md

```diff
@@ -0,0 +1,21 @@
+### End-to-End Workflow
+
+Each command in pbench-agent accepts the `--help` option and outputs a brief usage message
+
+The default set of tools for data collection can be enabled with
+
+	$pbench-register-tool-set
+
+To list all your registered tools
+
+	$pbench-list-tools
+
+You may then perform a built-in benchmark by running it's Pbench script
+
+	$pbench-user-benchmark – sleep 10
+
+The above command will collect data from the registered tools for the specified time period and save it in the `/var/lib/pbench-agent` directory.
+
+To move the results, the outcomes are tarred and sent to the configured pbench-server with
```

Comment:
Rather, if you want to process the results locally or on your own, you can; if you want to archive and process the results on a Pbench Server, you can use this command. Do you want to mention here that the move needs to be *authenticated* with an API Key? Probably not the place to get into details of generating and managing them, but maybe at least including the `--token <api-key>`??

Also, s/pbench-server/Pbench Server/